src/main/java/seedu/address/logic/commands/CreateEventCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LOCATION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Adds a member to the address book.
+ */
+public class CreateEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "createEvent";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates an event in CCACommander. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_LOCATION + "LOCATION "
+            + PREFIX_DATE + "DATE "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "Party "
+            + PREFIX_LOCATION + "Raffles Hall "
+            + PREFIX_DATE + "2023-10-16 ";
+
+    public static final String MESSAGE_SUCCESS = "New event added: %1$s";
+    public static final String MESSAGE_DUPLICATE_EVENT = "This event already exists in CCACommander. ";
+
+    private final Event toCreate;
+
+    /**
+     * Creates an CreateEventCommand to add the specified {@code Event}
+     */
+    public CreateEventCommand(Event member) {
+        requireNonNull(member);
+        toCreate = member;
```

Comment:
I think variable name should be renamed to event

Reply:
Yes I will fix this!

1. Yes I changed addEvent to createEvent for consistency throughout the execution flow of the createEventCommand.
2. The original getTypicalAddressBook only contained members, which was used by an original test case to check the correctness of loading members only. Since we have introduced events, the typicalAddressBook should contain both events and members. However, as the test cases are testing individually the loading of members and events, I thus created two more separate sample AddressBooks containing only members and events respectively. Currently getTypicalAddressBook is unused unless I create another test testing the loading of both together. Your thoughts on this?

Reply:
Hmm I think it will be better to either use the function or remove it since it's an unused function after all.

Reply:
Ok, I have removed it.

Reply:
Okay sounds good